bunq/sdk/json/geolocation_adapter.py

```diff
@@ -0,0 +1,61 @@
+from bunq.sdk.json import converter
+from bunq.sdk.model.generated import object_
+
+
+class GeolocationAdapter(converter.JsonAdapter):
+    # Field constants
+    _FIELD_LATITUDE = 'latitude'
+    _FIELD_LONGITUDE = 'longitude'
+    _FIELD_ALTITUDE = 'altitude'
+    _FIELD_RADIUS = 'radius'
+
+    @classmethod
+    def can_deserialize(cls):
+        """
+        :rtype: bool
+        """
+
+        return False
+
+    @classmethod
+    def deserialize(cls, target_class, obj):
+        """
+        :type target_class: float|type
```

Comment:
?

Reply:
That method does nothing besides raising an error that it's implemented. I guess it exists because it's overriding the parent classes method which shouldn't do anything?

Reply:
my remark was regarding the type annotation, see above.